src/test/java/org/gridsuite/geodata/server/GeoDataControllerTest.java

```diff
@@ -155,6 +158,18 @@ public void test() throws Exception {
                 .contentType(APPLICATION_JSON)
                 .content(toString(GEO_DATA_LINES)))
                 .andExpect(status().isOk());
+
+        mvc.perform(get("/" + VERSION + "/substations?networkUuid=" + networkUuid + "&variantId=" + VARIANT_ID + "&substationId=P1&substationId=P2")
+                        .contentType(APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(content().contentTypeCompatibleWith(APPLICATION_JSON))
+                .andExpect(jsonPath("$", hasSize(0)));
+
+        mvc.perform(get("/" + VERSION + "/lines?networkUuid=" + networkUuid + "&variantId=" + VARIANT_ID + "&lineId=NHV1_NHV2_2&lineId=NHV1_NHV2_1")
```

Comment:
Maybe you cloud add a test to try the other syntax for Mapping a Multi-Value Parameter as well :
"/" + VERSION + "/lines?networkUuid=" + networkUuid + "&variantId=" + VARIANT_ID + "&lineId=NHV1_NHV2_2,NHV1_NHV2_1"